docs/reference/commandline/network_ls.md

```diff
@@ -12,7 +12,7 @@ parent = "smn_cli"
 
     Usage:  docker network ls [OPTIONS]
 
-    Lists all the networks created by the user
+    List networks
```

Comment:
oh, yes, that's fine. was just a general note regarding Lists vs List, I don't think we ever decided what our "official" approach is there